students/wachulec_maria/lesson_10_organizing_files/selective_copy.py

```diff
@@ -0,0 +1,27 @@
+import os
+import shutil
+
+
+def solve_conflicts(file_name, new_file, extension):
+    copy_file_name = file_name
+    while copy_file_name in os.listdir(new_file):
+        copy_file_name = copy_file_name[:-len(extension)] + str(1) + extension
```

Comment:
why `str(1)` ? why not `'1'` 
Besides, what if even such file exist ?

Reply:
I wanted to use str(arg) and asked user about arg but I changed my mind. When is str(1) than it's easier to make this extension.